### DISCUSSION TITLE
Troubleshooting UART Connection with PC, FTDI-USB Converter, and Tinymovr R5.2

### DISCUSSION THREAD

**Discussion by yconst** [tinymovr/Tinymovr#318] [Q&A - Tinymovr] [answered]
How can I troubleshoot a UART connection issue between my PC, FTDI-USB converter, and Tinymovr R5.2 driver in Windows? I am using Realterm but I receive no response from the driver.

**Comment by yconst** [comment, marked as answer]
The connection between FTDI and Tinymovr should be like in the image below:
![IMG_6135](https://github.com/tinymovr/Tinymovr/assets/529837/cf32ab1c-fcef-4607-bfb1-5a30a29abc7a)

Note that the +3.3V rail is not connected to the FTDI adapter, as the adapter has its own power supply.

I've found that many terminal emulators have weird defaults that mess up communication. I've found this simple terminal emulator (termite) to be the most reliable in connecting, at least when using tinymovr

![Screenshot_2024-01-23_at_5 43 45_PM](https://github.com/tinymovr/Tinymovr/assets/529837/afe51485-4ab5-4c20-8a32-ce00f9fd18e3)

**Comment by mochex** [comment]
I use minicom in Linux and it works fine too. Keep in mind that for sending a `\n` you have to do with `ctrl + j`.
